package.json

```diff
@@ -19,13 +19,15 @@
   "homepage": "https://github.com/moeiscool/Shinobi#readme",
   "dependencies": {
     "body-parser": "^1.15.2",
+    "canvas": "^1.6.9",
```

Comment:
canvas probably shouldn't be here unless the machine will always have the libraries required to build it. Currently the provided installers do no install them for you.

Reply:
Agreed, I shouldn't have included my own package file in the push.  Sorry.